website/docs/r/cosmosdb_account.html.markdown

```diff
@@ -168,9 +168,7 @@ The following arguments are supported:
 
 * `name` - (Required) The capability to enable - Possible values are `AllowSelfServeUpgradeToMongo36`, `DisableRateLimitingResponses`, `EnableAggregationPipeline`, `EnableCassandra`, `EnableGremlin`, `EnableMongo`, `EnableTable`, `EnableServerless`, `MongoDBv3.4` and `mongoEnableDocLevelTTL`. 
```

Comment:
We should remove this section too.  We have more capabilities than listed here

Reply:
@sivethe thanks for your feedback. Could you provide the official documentation describing all the `capabilities`  here?

Reply:
I echo this